tests/PHPStan/Parser/TypeParserTest.php

```diff
@@ -393,7 +393,12 @@ public function provideParseData(): array
 				]),
 			],
 			[
-				'array{a: int, b: array{c: callable(): int}}',
+				'array{
+				    a: int, 
+				    b: array{
+				        c: callable(): int
+				    }
+				 }',
```

Comment:
During real usage, the lexer will receive this with a `* ` prefix on each line. We should add the prefix on this input, so that the test can detect any regressions related to this in the future.